package.json

```diff
@@ -36,6 +36,8 @@
     "debug": "node --inspect-brk ./lighthouse-cli/index.js",
     "start": "yarn build-report --standalone && node ./lighthouse-cli/index.js",
     "jest": "node --experimental-vm-modules ./node_modules/jest/bin/jest.js",
+    "____comment": "echo https://github.com/cucumber/cucumber-js/issues/1844",
```

Comment:
delete?

Reply:
It's addressing a really weird workaround directly below it.